hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/statemachine/DatanodeConfiguration.java

```diff
@@ -202,6 +202,21 @@ public class DatanodeConfiguration extends ReconfigurableConfig {
   private Duration blockDeleteCommandWorkerInterval =
       BLOCK_DELETE_COMMAND_WORKER_INTERVAL_DEFAULT;
 
+  /**
+   * The maximum count of recent transactions cached in memory. This cache
+   * prevents duplicate transactions from being written to the database.
+   * By default, the value is set to the same as
+   * {@link org.apache.hadoop.hdds.scm.ScmConfig#blockDeletionLimit}.
```

Comment:
org.apache.hadoop.hdds.scm.ScmConfig#blockDeletionLimit  current it's default value is 20000 now.